scraper.py

```diff
@@ -0,0 +1,171 @@
+import selenium
+import re
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.by import By
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from webdriver_manager.chrome import ChromeDriverManager
+# Q & To do
+# How to put matter to json, put functions in for loop as well?
+# Style requirements
+# Remove all the held from the minutes item
+# Put the matters in the minutes along with the Titles
+# Decision Constant
+# Person In the same script or ?
+# get different meetings in the wannted time period
+
+driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()))
+
+driver.get("https://atlantacityga.iqm2.com/Citizens/SplitView.aspx?Mode=Video&MeetingID=3587&Format=Minutes")
+
+body_name = driver.find_element(By.ID, "ContentPlaceHolder1_lblMeetingGroup").text #body name 
+video_link = driver.find_element(By.ID, "MediaPlayer1_html5_api").get_attribute("src") # video link (mp4)
+
+titles = driver.find_elements(By.CSS_SELECTOR,"td[class='Title'][colspan='10']")
+minute = "" # minutes name
+for title in titles:
+    try:
+        if (len(title.text)!= 0):
+            minute = title.text
+            print(minute)
+    except selenium.common.exceptions.NoSuchElementException:
+        pass
+
+matter_type = "" # matter type
+matter_name = "" # matter name 
+matter_title = "" # matter title
+sponsor = "" #sponsor for the matter not done yet
+decision = "" # decsion for the matter
+v_yes = "" # people voted yes
+v_no = "" # people voted no
+
+matters = driver.find_elements(By.CSS_SELECTOR,"td[class='Title'][colspan='9']")
+for matter in matters:
+    try:
```

Comment:
I would put this entire try and except block in a function called `parse_single_matter` that has something like the following definition and usage:

```python
def parse_single_matter(html_matter_head_element: Element) -> ingestion_models.Matter:
    # something here
    
# usage
matter_elements = driver.find_elements("parameters...")
parsed_matter_models = []
for matter_element in matter_elements:
    matter_model = parse_single_matter(matter_element)
    # assume None means that it was a "held matter" or something else went wrong that we don't want to store it
    if matter_model is not None:
        parsed_matter_models.append(matter_model)
  
```